dotcom-rendering/docs/architecture/028-dotenv.md

```diff
@@ -38,3 +38,27 @@ const secrets = [
     // ...
 ];
 ```
+
+## Security
+
+`.env` variables are only available on the server, you should ensure that you **never** pass an environment variable into the clientside code
+e.g
+
+```jsx
+<Island>
+	<!-- Bad :((( -->
+	<MyComponent myProp={process.env.MY_VAR} />
+</Island>
+```
+
+This has considerable security implementations as this variable would then be available to any consumer of the site.
+
+## Limitations
```

Comment:
Do we have any here?

Reply:
I'm not sure I understand the question!

Reply:
Apologies, I didn't realise that this is a heading for the next points...